docs-ref-conceptual/includes/cli-install-linux-apt.md

```diff
@@ -59,7 +57,7 @@ If you prefer a step-by-step installation process, complete the following steps
     sudo chmod go+r /etc/apt/keyrings/microsoft.gpg
     ```
 
-3. Add the Azure CLI software repository:
+3. <div id="set-release"/>Add the Azure CLI software repository:
```

Comment:
Revert #3505 to get rid of `Line 97, Column 93: [Warning: bookmark-not-found] Cannot find bookmark '#set-release' in 'docs-ref-conceptual/includes/cli-install-linux-apt.md'.` warning.